advanced-achievements-plugin/src/test/java/com/hm/achievement/command/pagination/CommandPaginationTest.java

```diff
@@ -10,6 +10,7 @@
 import org.bukkit.ChatColor;
 import org.junit.Before;
 import org.junit.ClassRule;
+import org.junit.Ignore;
```

Comment:
All the exiting test classes should probably be reverted, changing the import ordering or adding unused imports creates commit noise.